src/components/molecules/cards/responsive-cards.tsx

```diff
@@ -1,29 +1,28 @@
-import React from "react";
+import React, {ReactElement, useMemo} from "react";
+import { LayoutItem } from "react-grid-layout";
 
 import { ResponsiveGrid } from "../../atoms/responsive-grid/responsive-grid";
-import { Card } from "../../atoms/card/card";
 
-export const ResponsiveCards = () => {
-  const layouts = {
-    lg: [
-      { i: '1', x: 0, y: 0, w: 1, h: 1, static: false },
-    ],
-  };
+import { UICard } from "../../../modules/ui-room-builder/store/slice";
 
-  const cards = [
-    { title: 'a', key: '1' },
-  ];
+interface Props {
+  cards: UICard[];
+  onLayoutChange: (x: typeof LayoutItem[]) => void;
+  renderCard: (item: UICard, index: number) => ReactElement,
+}
+
+export const ResponsiveCards = ({ cards, onLayoutChange, renderCard }: Props) => {
```

Comment:
The component shouldn't really be connected to redux state. It has to operate with props only and be somewhat reusable or at least no tightly coupled on an implementation